e2e/testsuite/testsuite.go

```diff
@@ -383,7 +383,7 @@ func GetIBCToken(fullTokenDenom string, portID, channelID string) transfertypes.
 // the test. If the test is running in CI, more nodes are used, when running locally a single node is used by default to
 // use less resources and allow the tests to run faster.
 // both the number of validators and full nodes can be overwritten in a config file.
-func getValidatorsAndFullNodes(chainIdx int) (int, int) {
+func getValidatorsAndFullNodes(chainIdx int) (vailidators int, fullNodes int) {
```

Comment:
[q] is this a linting change? I generally dislike named return arguments since they initialize variables outisde of the function body. Does a linter require this?

Reply:
no, but I can swear there was some reason I made the change. After I get main merged, I'll try and figure out why.

Reply:
@chatton after reviewing, I felt that this was clearer with the named returns.